packages/maskbook/src/plugins/BarnBridge/apis/index.ts

```diff
@@ -84,15 +84,12 @@ function PrettifyLiquidity(liquidity: string) {
 }
 
 export async function SYGetPortfolio(walletAddress: string) {
-    const response = await fetch(
-        `https://api-v2.barnbridge.com/api/smartyield/users/${walletAddress}/portfolio-value`,
-        {
-            body: null,
-            method: 'GET',
-            mode: 'cors',
-            credentials: 'omit',
-        },
-    )
+    const response = await fetch(urlcat(API_URL, '/api/smartyield/pools', { originator: 'all' }), {
```

Comment:
move `/api/` prefix string to `API_URL` constant, thanks

Reply:
moved